src/AdminConsole/Services/ISetupService.cs

```diff
@@ -0,0 +1,6 @@
+namespace Passwordless.AdminConsole.Services;
+
+public interface ISetupService
+{
+    Task<bool> HasSetupCompletedAsync();
```

Comment:
```suggestion
    Task<bool> IsSetupCompletedAsync();
```

Reply:
Bad grammar.

Reply:
I know, but I find this to be more common in programming contexts.